plugins/gatsby-theme-iterative-docs/normalize-sidebar.js

```diff
@@ -122,12 +82,10 @@ function normalizeItem({ rawItem, parentPath, resultRef, prevRef }) {
   }
 }
 
-function normalizeSidebar({
+function normalizeSidebar(
   data,
-  parentPath,
-  parentResultRef,
-  startingPrevRef
-}) {
+  { parentPath = '', parentResultRef, startingPrevRef } = {}
+) {
```

Comment:
Small tweak to the signature to make this function more natural to use IMO, the rest of the args are just used to recursive calls so that function's practically unary.